api/app/schemas/morecast_v2.py

```diff
@@ -0,0 +1,52 @@
+""" This module contains pydantic models for Morecast v2"""
+
+from enum import Enum
+from pydantic import BaseModel
+
+
+class ModelChoice(str, Enum):
+    """ Enumerator for all valid forecasted value types """
+    GDPS = 'GDPS'
```

Comment:
Can we add GFS?

Reply:
Added in: https://github.com/bcgov/wps/pull/2643/commits/e859b46c845f44d7a568543592502f28f5aff4e0